acceptanceTests/testsUtils.go

```diff
@@ -103,6 +111,90 @@ func getServiceExternalIp(ctx context.Context, namespace string, service string)
 	return externalIp, nil
 }
 
+func switchKubeContextForTest(t *testing.T, newContextName string) error {
+	prevKubeContextName, err := getKubeCurrentContextName()
+	if err != nil {
+		return err
+	}
+
+	if err := setKubeCurrentContext(newContextName); err != nil {
+		return err
+	}
+
+	t.Cleanup(func() {
+		err := setKubeCurrentContext(prevKubeContextName)
```

Comment:
can be changed to single line
if ; err != nil

Reply:
done (by Roee)